<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LastForm</class>
 <widget class="QWidget" name="LastForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>426</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>333</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Оформление заказа</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>258</width>
     <height>419</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Заполните поля</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Имя</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="name_edit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Фамилия</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="surname_edit"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Отчество(Если имеется)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="patr_edit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="phone_label">
      <property name="text">
       <string>Номер телефона</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLineEdit" name="phone_edit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="email_label">
      <property name="text">
       <string>Электронная почта</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QLineEdit" name="email_edit"/>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Комментарий (Не обязательно)</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QTextEdit" name="com_edit"/>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="final_button">
      <property name="text">
       <string>Оформить заказ</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
